demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1169,15 +1192,77 @@ export class DemoMeetingApp
       }
       const tileId = videoTile.id();
       const tileIndex = this.tileIdToTileIndex[tileId];
-      this.getStats(tileIndex);
       if (tileState.localTile) {
-        this.statsCollector.showUpstreamStats(tileIndex);
+        this.showVideoStats(tileIndex, this.videoUpstreamMetricsKeyStats, videoMetricReport[tileState.boundAttendeeId], 'Upstream');
       } else {
-        this.statsCollector.showDownstreamStats(tileIndex);
+        this.showVideoStats(tileIndex, this.videoDownstreamMetricsKeyStats, videoMetricReport[tileState.boundAttendeeId], 'Downstream');
       }
     }
   }
 
+  showVideoStats = (
```

Comment:
Can we separate this in the existing `WebRTCStatsCollector`? I see you are removing it and adding the code here, since our `meetingV2.ts` is already big enough 😅  and stats collection can actually be separated out.

Reply:
Oh since I already reduced a lot code in WebRTCStatsCollector and I thought we don't need a seperate controller to handle this part of logic. Actually I don't see any other separate controllers. So I move this part in the meetingV2.ts. So does it really make and difference if I separate this?

Reply:
Yeah, it does not make difference functionality wise but we should be separating out components until now we have not practiced this but I thought good to keep separation here, we separated the video processor classes in the demo. Otherwise, our meetingV2 is going to get larger and larger. Also, I think, it makes it little easier to ready through the class which are logically abstracted out / separated.

Reply:
There is really only one function left so I don't think we need to separate them out. The existing WebRTCStatsCollector should be deleted anyway since the demo no longer collects stats themselves.

Reply:
That said, maybe we could do a subsequent PR to move all stats-related functions to a new file.